core/src/Data/Query/CountAllQuery.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+namespace Harmony\Core\Data\Query;
+
+class CountAllQuery extends CountQuery {
```

Comment:
I think this classs could make no sense since count is count... what's a countAll?